src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +425,180 @@ public void SetFocus(bool focused)
+        public bool SetViewScale (double scaleFactor, Point scrollTo)
+        {
+            return Interop.ChromiumEwk.ewk_view_scale_set(_realHandle, scaleFactor, scrollTo.X, scrollTo.Y);
+        }
+
+        /// <summary>
+        /// Sets the current page's visibility.
+        /// </summary>
+        /// <param name="enable">'true' to set on the visibility of the page, 'false' otherwise.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetViewVisibility (bool enable)
+        {
+            return Interop.ChromiumEwk.ewk_view_visibility_set(_realHandle, enable);
+        }
+
+        /// <summary>
+        /// Get and Sets the scroll position of the page.
+        /// </summary>
+        /// <returns>The class Point object with X, Y coordinates.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Point ScrollPos
+        {
+            get
+            {
+                Point p;
+                Interop.ChromiumEwk.ewk_view_scroll_pos_get(_realHandle, out p.X, out p.Y);
+                return p;
+            }
+            set
+            {
+                Interop.ChromiumEwk.ewk_view_scroll_set(_realHandle, value.X, value.Y);
+            }
+        }
+
+        /// <summary>
+        /// Scrolls the webpage of view by x and y.
+        /// </summary>
+        /// <param name="scrollBy">The class Point object with X, Y coordinates.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ScrollBy (Point scrollBy)
+        {
+            Interop.ChromiumEwk.ewk_view_scroll_by(_realHandle, scrollBy.X, scrollBy.Y);
+        }
+
+        /// <summary>
+        /// Searches and highlights the given text string in the document.
+        /// </summary>
+        /// <param name="text">The text to find.</param>
+        /// <param name="option">The options to find.</param>
+        /// <param name="maxMatchCount">The maximum match count to find, unlimited if 0.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool FindText (string text, FindOption option, int maxMatchCount)
+        {
+            return Interop.ChromiumEwk.ewk_view_text_find(_realHandle, text, option, maxMatchCount);
+        }
+
+        /// <summary>
+        /// Requests loading of the given request data.
+        /// </summary>
+        /// <param name="url">The uniform resource identifier to load.</param>
+        /// <param name="httpMethod">The http method.</param>
+        /// <param name="httpHeaders">The http headers.</param>
+        /// <param name="httpBody">The http body data.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetUrlRequest (string url, HttpMethod httpMethod, IDictionary<string, string> httpHeaders, string httpBody)
```

Comment:
It should be void, in C# we don't return boolean on success.

Reply:
Done